helpers/utils.go

```diff
@@ -13,3 +17,12 @@ func GetSignatureFromReferenceAndPubKey(reference, publicKey string) string {
 	hash.Write([]byte(concat))
 	return fmt.Sprintf("%x", hash.Sum(nil))
 }
+
+// GetContextValue to extract the request-id value from the context passed
+func GetContextValue(ctx context.Context, k model.Key) (string, error) {
+	if v := ctx.Value(k); v != nil {
+		fmt.Println("found value:", v)
+		return fmt.Sprintf("%s", v), nil
+	}
+	return "", errors.New("something-went-wrong")
```

Comment:
please check how we are getting it in api-service. `a.getRequestID` or so.

Reply:
The update has been made to always pass request-id from the consumer app, and also in api-service, and doing same here might be redundant.

Also, I thought not to introduce another dependency overhead.